examples/relative_free_energy.py

```diff
@@ -1,194 +1,63 @@
 import argparse
 import sys
 
-import matplotlib.pyplot as plt
 import numpy as np
-import pymbar
 from rdkit import Chem
-from rdkit.Chem import AllChem, Draw
 
-from timemachine.constants import BOLTZ
 from timemachine.fe import atom_mapping, pdb_writer
+from timemachine.fe.rbfe import HostConfig, estimate_relative_free_energy, plot_atom_mapping_grid
 from timemachine.fe.single_topology_v3 import SingleTopologyV3
-from timemachine.fe.system import convert_bps_into_system
-from timemachine.fe.utils import get_romol_conf
 from timemachine.ff import Forcefield
-from timemachine.ff.handlers import openmm_deserializer
-from timemachine.lib import LangevinIntegrator, MonteCarloBarostat, custom_ops
-from timemachine.md import builders, minimizer
-from timemachine.md.barostat.utils import get_bond_list, get_group_indices
+from timemachine.md import builders
 from timemachine.testsystems.relative import get_hif2a_ligand_pair_single_topology
 
 
-def sample(bound_impls, masses, x0, v0, box0, lamb, group_idxs, seed, n_frames=500, burn_in=10000):
+def write_trajectory_as_pdb(mol_a, mol_b, core, all_frames, host_topology, prefix):
 
-    dt = 1e-3
+    dummy_st = SingleTopologyV3(mol_a, mol_b, core, None)
 
-    temperature = 300.0
-    friction = 1.0
-
-    intg = LangevinIntegrator(temperature, dt, friction, masses, seed)
-    intg_impl = intg.impl()
-
-    baro = MonteCarloBarostat(len(masses), 1.0, temperature, group_idxs, 15, seed + 1)
-    baro_impl = baro.impl(bound_impls)
+    for window_idx, window_frames in enumerate(all_frames):
+        out_path = f"{prefix}_{window_idx}.pdb"
+        writer = pdb_writer.PDBWriter([host_topology, dummy_st.mol_a, dummy_st.mol_b], out_path)
+        for frame in window_frames:
+            host_frame = frame[: host_topology.getNumAtoms()]
+            ligand_frame = frame[host_topology.getNumAtoms() :]
+            mol_ab_frame = pdb_writer.convert_single_topology_mols(ligand_frame, dummy_st)
+            writer.write_frame(np.concatenate([host_frame, mol_ab_frame]) * 10)
+    writer.close()
 
-    # tbd: add barostat
-    ctxt = custom_ops.Context(x0, v0, box0, intg_impl, bound_impls, baro_impl)
 
-    # burn-in
-    ctxt.multiple_steps_U(
-        lamb=lamb,
-        n_steps=burn_in,
-        lambda_windows=[lamb],
-        store_u_interval=0,
-        store_x_interval=0,
-    )
+def run_pair(mol_a, mol_b, core, forcefield, n_frames, protein_path):
```

Comment:
Worth making `seed` and argument to be consistent with other examples?

Reply:
sure